docs/appendices/release-notes/unreleased.rst

```diff
@@ -57,6 +57,10 @@ Changes
 Fixes
 =====
 
+- Fixed a regression introduced in 4.2.0 that resulted in ``NULL`` values being
+  returned for columns used in the ``PARTITIONED BY`` clause instead of the
```

Comment:
```suggestion
  returned for partitioned by columns instead of the
```

Not sure which is easier to understand?

Reply:
for me, the original variant is easier to read